web/frontend/styles/casebook.scss

```diff
@@ -295,14 +295,16 @@ form.edit_content_resource, form.edit_content_section, form.edit_content_caseboo
     margin-bottom: 2px;
     padding: 20px 20px 20px 30px;
     background-color: $white;
+    display: flex;
+    justify-content: space-between;
+
     .casebook-title {
       @include sans-serif($bold, 20px, 24px);
       color: $dark-gray;
     }
-    .reading-time {
```

Comment:
This appeared to be unused—`reading-time` only appears on the section pages, and styles for that are defined further down in the file.